chapters/api_concepts.txt

```diff
@@ -312,16 +318,15 @@ The following infos of objects can be queried:
 | description   | `STRING`         |    No    | explanation of the object, e.g., for a tooltip
 | parameter     | `PARAMETER_LIST` |    Yes   | list of supported parameters (array of `ANARIParameter`)
```

Comment:
Wasn't `anariGetObjectInfo` also meant to inspect device parameters (with `objectType=DEVICE` and `objectSubtype=objectSubtype`)? Can that still be done now after switching to device handles? I.e., can the function be used without previously committing the device? Because some devices can only be committed once, so the parameters need to be set before.

Reply:
> Can that still be done now after switching to device handles? I.e., can the function be used without previously committing the device? 

Yes, I think this is the clearest use of the function. `anariCommitParameters()` is only relevant to committing parameters, so we don't have any other semantics to worry about -- "committing the device" doesn't mean anything else.